src/feedback/trulens/feedback/prompts.py

```diff
@@ -9,9 +9,9 @@
 Please answer using the entire template below.
 
 TEMPLATE:
-Score: <The score based on the given criteria>
 Criteria: <Provide the criteria for this evaluation>
 Supporting Evidence: <Provide your reasons for scoring based on the listed criteria step by step. Tie it back to the evaluation being completed.>
+Score: <The score based on the given criteria>
```

Comment:
Can you try to put some examples where scoring improves before and after this change?